module2/src/com/epam/arraysofarrays/MatrixWorkerOrderN.java

```diff
@@ -0,0 +1,69 @@
+package com.epam.arraysofarrays;
+
+public class MatrixWorkerOrderN {
+    //task 4
+    public static  int[][] createSquareMatrixOrderPositiveN (int positiveN) {
```

Comment:
It is better to rename e.g. 
```suggestion
    public static int[][] buildMatrixAccordingToPattern1(int n) {
```